### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by brishtibheja**
I was about to add another commit but you merged the PR.

### CODE REVIEW

File: src/searching.md [hunk 1]
```diff
@@ -382,7 +382,7 @@ you need to tell Anki not to treat them specially. This is called "escaping a ch
   `part:"after the colon"`.
 
 - `And`/`Or`\
-  To search for the words `or` and `and`, wrap them with double quotes. For example, `dog "and" cat` searches for "dog", "cat" and the word "and".
+  To search for the words "or" and "and", wrap them with double quotes. For example, `dog "and" cat` searches for "dog", "cat" and the word "and".
```
Comment (dae) [inline src/searching.md:385]:
```suggestion
  To search for the words `or` and `and`, wrap them with double quotes. For example, `dog "and" cat` searches for "dog", "cat" and the word "and".
```

Comment (brishtibheja) [reply src/searching.md:385]: I understand your reasoning but double quotes are used as we're talking about the string of characters *to search* and not the search term itself even though we say "wrap them in double quotes".

Comment (dae) [reply src/searching.md:385]: I'm afraid I find my suggested version easier to read than what you currently have.

Screenshot 2024-09-07 at 4 13 50 PM

Comment (brishtibheja) [reply src/searching.md:385]: "to search for these words"? it's in a list, shouldn't confuse people.

Comment (dae) [reply src/searching.md:385]: Works for me.

**Pull request merged**